packages/datastore/amplify_datastore/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -662,6 +771,88 @@
 			};
 			name = Profile;
 		};
+		4B0480EB28DE65EA00EF3F1F /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = B23E4E08D535BDEC9129FF3F /* Pods-RunnerTests.debug.xcconfig */;
+			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				BUNDLE_LOADER = "$(TEST_HOST)";
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
+				GCC_C_LANGUAGE_STANDARD = gnu11;
+				GENERATE_INFOPLIST_FILE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
Should `IPHONEOS_DEPLOYMENT_TARGET` be set to the minimum supported iOS version to run integ tests?

Reply:
Updated 👍